CHANGES.rst

```diff
@@ -7,7 +7,7 @@ Changelog
     file is managed by towncrier. You *may* edit previous change logs to
     fix problems like typo corrections or such.
     To add a new change log entry, please see
-    https://docs.pulpproject.org/en/3.0/nightly/contributing/git.html#changelog-update
+    https://docs.pulpproject.org/pulpcore/contributing/git.html#changelog-update
```

Comment:
the link you provided is the latest released,
this is the one that is built nightly:
https://docs.pulpproject.org/pulpcore/en/master/nightly/contributing/git.html#changelog-update

Reply:
Yes, I know that this is for the latest version released. I thought that `/en/3.0/nightly/` should be removed as a whole since it was placed everywhere where we talk about `pulpcore`. Was it on purpose?

Reply:
you are right about removing it since the link is broken, 
I believe the change would be
```
/3.0/master
```
nightly build is better because the documentation could be on master and not on the latest release